### PR TITLE
Muting mk2

### PR DESCRIPTION
**Pull request by Simber1**

### CODE REVIEW

**Comment by Simber1** [comment]
Doesn't work if the muted users json is empty

File: commands/mute.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+module.exports = {
+	name: 'mute',
+    description: 'Muting users',
+    usage: 'mute',
+	execute(message, args) {
+        const fs = {readFileSync, writeFile} = require ('fs');
+
+
+        if (message.member.roles.has(roleIDs.mod)){
```
Comment (Danial-Q) [inline commands/mute.js]: Can omit this by just throwing a modOnly: true above execute

File: commands/mute.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+const {roleIDs} = require('../config.json');
+
+module.exports = {
+	name: 'mute',
+    description: 'Muting users',
+    usage: 'mute',
+	execute(message, args) {
+        const fs = {readFileSync, writeFile} = require ('fs');
```
Comment (Danial-Q) [inline commands/mute.js]: const {readFileSync, writeFile} = require ('fs');

Comment (Simber1) [reply commands/mute.js]: That wasn't working.

**Review by Danial-Q — APPROVED** [review]

**Pull request merged**